pkg/update/update_cockroach_version_common.go

```diff
@@ -86,7 +86,7 @@ func makeIsCRBPodIsRunningNewVersionFunction(
 			return statusError
 		} else if err != nil { // this is an error
 			// TODO uncertain why this is logging is throwing an error
-			l.Error(err, "error getting pod")
+			// l.Error(err, "error getting pod")
```

Comment:
I commented this out in another PR and the merge must have gotten messed up.  https://github.com/cockroachdb/cockroach-operator/issues/532 is open to deal with this.